src/mono/nuget/Microsoft.NET.Runtime.WebAssembly.Sdk/Sdk/AutoImport.props

```
@@ -1,5 +1,5 @@
 <Project>
   <PropertyGroup>
-    <WasmNativeWorkload>true</WasmNativeWorkload>
+    <WasmNativeWorkloadAvailableList>+net6.0+net7.0+net8.0+</WasmNativeWorkloadAvailableList>
```

```suggestion
    <WasmNativeWorkloadAvailableList>+net6.0+net7.0+net8.0+</WasmNativeWorkloadAvailableList>
```
copy what release/7.0 does here not this